utils/build.ps1

```diff
@@ -2188,6 +2174,12 @@ function Get-CompilersDefines([Hashtable] $Platform, [string] $Variant, [switch]
     $SwiftFlags += @("-use-ld=lld");
   }
 
+  $CMakeStaticLibPrefixSwiftDefine = if ((Get-PinnedToolchainVersion) -eq "0.0.0") {
+    @{ CMAKE_STATIC_LIBRARY_PREFIX_Swift = "lib"; }
+  } else {
+    @{}
+  }
```

Comment:
Why is this needed? The only place that this should be needed is for the `Build-Compilers` and we should pass it there.